libs/langchain/langchain/schema/runnable/base.py

```diff
@@ -1063,57 +1060,53 @@ def stream(
         else:
             run_manager.on_chain_end(final)
 
-    async def astream(
+    async def _atransform(
         self,
-        input: Input,
-        config: Optional[RunnableConfig] = None,
-        **kwargs: Optional[Any],
+        input: AsyncIterator[Input],
+        run_manager: AsyncCallbackManagerForChainRun,
+        config: RunnableConfig,
     ) -> AsyncIterator[Output]:
-        from langchain.callbacks.manager import AsyncCallbackManager
-
-        # setup callbacks
-        config = config or {}
-        callback_manager = AsyncCallbackManager.configure(
-            inheritable_callbacks=config.get("callbacks"),
-            local_callbacks=None,
-            verbose=False,
-            inheritable_tags=config.get("tags"),
-            local_tags=None,
-            inheritable_metadata=config.get("metadata"),
-            local_metadata=None,
-        )
-        # start the root run
-        run_manager = await callback_manager.on_chain_start(dumpd(self), input)
-
         steps = [self.first] + self.middle + [self.last]
         streaming_start_index = len(steps) - 1
 
         for i in range(len(steps) - 1, 0, -1):
-            if type(steps[i]).transform != Runnable.transform:
+            if type(steps[i]).atransform != Runnable.atransform:
```

Comment:
I guess this was incorrect before?

Reply:
I think so? Wouldn't make sense to compare to the non-async version here